test/services/sentiment_analysis_service_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'test_helper'
+
+class SentimentAnalysisService < ActiveSupport::TestCase
+  test "#determine_sentiment" do
+    assert_nil SentimentAnalysisService.determine_sentiment()
+    assert_nil SentimentAnalysisService.determine_sentiment(nil, nil, nil)
+
+    cases = [
+      { input: ["simply the worst"], expectation: Sentiment::Negative }
+      { input: ["simply the worst", "abhorant"], expectation: Sentiment::StrongNegative }
+      { input: ['pizza', "fries"], expectation: Sentiment::Neutral }
+      { input: ["only the best", "simply the worst"], expectation: Sentiment::Mixed }
+      { input: ["only the best"], expectation: Sentiment::Positive }
+      { input: ["Awesome possum", 'the best'], expectation: Sentiment::StrongPositive }
+    ]
+
+    cases.each do |test_case|
```

Comment:
sr: Test do a nice job of explaining the method. These should be easy to extend.